src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,29 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
```

Comment:
The method name 'getEmployByOrder' contains a typo. Consider renaming it to 'getEmployeesByOrder' to correctly reflect the action performed and the entities it operates on.